server/db/db_connect.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017 The Absolute Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const mongoose = require('mongoose');
+
+const con = mongoose.connection;
+
+function connectServer() {
```

Comment:
This function should be located in common/ directory?
Also, this function can be called by any modules.